src/code/z_map_exp.c

```diff
@@ -1,19 +1,26 @@
 #include "global.h"
 
-s16 sMinimapInitPosX = 0;
-s16 sMinimapInitPosZ = 0;
-s16 sMinimapInitDir = 0;
-
-s32 sDungeonAndBossSceneIds[] = {
-    SCENE_MITURIN,    SCENE_HAKUGIN,    SCENE_SEA,    SCENE_INISIE_N,  SCENE_INISIE_R,
-    SCENE_MITURIN_BS, SCENE_HAKUGIN_BS, SCENE_SEA_BS, SCENE_INISIE_BS,
-};
+s16 sPlayerInitPosX = 0;
+s16 sPlayerInitPosZ = 0;
+s16 sPlayerInitDir = 0;
 
 /**
- * If the current scene is a dungeon or boss scene, this function returns an index
- * indicating which one it is. Otherwise, it returns -1.
+ * Gets the DungeonSceneIndex for the current scene
+ * @return the current scene's DungeonSceneIndex, or -1 if it isn't a dungeon or boss scene.
  */
-s32 Map_GetDungeonOrBossAreaIndex(PlayState* play) {
+s32 MapExp_TryGetDungeonSceneIndex(PlayState* play) {
```

Comment:
```suggestion
s32 Map_GetDungeonSceneIndex(PlayState* play) {
```
Try doesn't really add anything imo. Also this is the only function that has `MapExp` as the prefix